lib/sprockets/manifest.rb

```diff
@@ -129,6 +129,23 @@ def find(*args)
       nil
     end
 
+    # Public: Find the source of assets by paths.
+    #
+    # Returns Enumerator of assets file content.
+    def find_source(*args)
```

Comment:
@rafaelfranca @djoume shouldn't it be named something like `find_sources` as it returns/yields multiple sources, or have it working with only one asset? I see the use case for inline SVG images but I believe working on one asset at a time to be a more common use case.